src/Sdk.Tests/AzurePipelines/Util/LocalFileProvider.cs

```diff
@@ -0,0 +1,54 @@
+namespace Runner.Server.Azure.Devops
+{
+    public class LocalFileProvider : IFileProvider // TODO: Fix IFileProvider namespacing
```

Comment:
There are a few files that don't have a namespace defined in the SDK, which really should be adjusted to namepace `Server.Runner.Azure.Devops` for consistency:

- src/Sdk/AzurePipelines/AzureDevops.cs
- src/Sdk/AzurePipelines/IFileProvider.cs
- src/Sdk/AzurePipelines/IVariablesProvider.cs

Reply:
I can open another issue and this can be addressed separately?